sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/sonarqube.yaml

```diff
@@ -0,0 +1,121 @@
+provider:
+  metadata:
+    name: SonarQube
+    category: Code analysis
+    references:
+      - description: OWASP Top 10 2021 Category A7 - Identification and Authentication Failures
+        link: https://owasp.org/Top10/A07_2021-Identification_and_Authentication_Failures/
+        type: Standards
+      - description: Google Cloud - Creating and managing service account keys
+        link: https://cloud.google.com/iam/docs/keys-create-delete
+        type: Documentation
+      - description: OWASP Top 10 2017 Category A3 - Sensitive Data Exposure
+        link: https://www.owasp.org/www-project-top-ten/2017/A3_2017-Sensitive_Data_Exposure
+        type: Standards
+      - description: MITRE, CWE-798 - Use of Hard-coded Credentials
+        link: https://cwe.mitre.org/data/definitions/798
+        type: Standards
+      - description: MITRE, CWE-259 - Use of Hard-coded Password
+        link: https://cwe.mitre.org/data/definitions/259
+        type: Standards
+      - description: SANS Top 25 - Porous Defenses
+        link: https://www.sans.org/top25-software-errors/#cat3
+        type: Standards
```

Comment:
Most of these were copied directly from a Google secret detection. It appears that one of the links here is Google-specific so I'll remove it.

It would make a lot of sense if the standards could be shared between all rules, and that we only use the references to link to documentation on generating/rotating keys.